src/lib/parser.c

```diff
@@ -1171,6 +1175,42 @@ static RdbStatus zsetZiplistItem(RdbParser *p, BulkInfo *ziplistBulk) {
     return RDB_STATUS_OK;
 }
 
+static RdbStatus unzipStreamListpack(RdbParser *p, char *nodekey, unsigned char *lp) {
+    UNUSED(nodekey);
+    streamIterator si;
+    streamIteratorStart(&si,nodekey,lp);
+    RdbStreamID id;
+    int64_t numfields;
+
+    while(streamIteratorGetID(&si,(streamID *) &id,&numfields)) {
+        while(numfields--) {
+
+            unsigned char *field, *value;
+            int64_t field_len, value_len;
+            streamIteratorGetField(&si,&field,&value,&field_len,&value_len);
+
+            EmbeddedBulk embField, embValue;
+            if (!allocEmbeddedBulk(p, field, field_len, 0, &embField))
+                return RDB_STATUS_ERROR;
+
+            if (!allocEmbeddedBulk(p, value, value_len, 0, &embValue))
+                return RDB_STATUS_ERROR;
+
+            registerAppBulkForNextCb(p, &embField.binfo);
+            registerAppBulkForNextCb(p, &embValue.binfo);
+            CALL_HANDLERS_CB(p,
+                             restoreEmbeddedBulk(p, &embField);restoreEmbeddedBulk(p, &embValue);,
```

Comment:
```suggestion
                             restoreEmbeddedBulk(p, &embField);restoreEmbeddedBulk(p, &embValue),
```
:cold_sweat: Macros are bit hard to read and debug, took me a while to understand the syntax. Last `;` is extra here.